app/lib/actions/pulp3/repository/remove_all_units.rb

```diff
@@ -0,0 +1,18 @@
+module Actions
+  module Pulp3
+    module Repository
+      class RemoveAllUnits < Pulp3::AbstractAsyncTask
```

Comment:
Where do you use this action? I see `remove_all_content` being used but not through `RemoveAllUnits`

Reply:
It _was_ used at one point to work around an issue in pulp_rpm. But it no longer is used. We should remove it!